### DISCUSSION TITLE
Unity project shows as Shaderlab in language breakdown, linguist-generated not working

### DISCUSSION THREAD

**Discussion by juuunoz** [github-linguist/linguist#5979] [Classification] [answered]
I'm trying to get my Unity project to show as C# language, but the majority of the language turns out to be ShaderLab

I've tried using linguist-generated to ignore it, like this

Assets/TextMesh?Pro/* linguist-generated

but the main language still shows up as ShaderLab.

here's the repo: https://github.com/nightstraps/roguelite-typer

**Comment by lildude** [comment, marked as answer]
> Assets/TextMesh?Pro/* linguist-generated

This will only apply to files directly in that directory and not the files within the directories in this directory. You need to use `**` to recurse into the folders too. So...

`Assets/TextMesh?Pro/** linguist-generated`

... will do the trick.

**Reply by juuunoz** [reply]
it works now. thank you!
